add-on/chrome/bootstrap.js

```diff
@@ -604,24 +604,25 @@ var WindowListener = {
         }
 
         let browser = gBrowser.selectedBrowser;
-
-        let cursorContainer = document.getElementById("loop-remote-cursor-container");
-        if (!cursorContainer) {
-          cursorContainer = document.createElement("img");
+        let cursor = document.getElementById("loop-remote-cursor");
+        if (!cursor) {
+          // Create a 100% size container to keep the pointer inside.
+          // This allows us to hide the overflow when out of bounds.
```

Comment:
It's not clear why the container is 100%.  Please be more explicit that this occurs because the container is a stack and we have set no other contraints on the div.